lib/py/cli.py

```diff
@@ -11,14 +11,18 @@ def inf(opts):
     if opts['in']:
         return open(opts['in'], 'rb')
     else:
-        return sys.stdin
+        if sys.stdin.isatty():
+            return sys.stdin
+        return sys.stdin.buffer
```

Comment:
This will not work with python 2, will it? Should we force to use python3?

Reply:
See #1405

Reply:
So we decide that we want to work with python2, but this code will not work with it:

$ python2 -c 'import sys; print sys.stdin.buffer'Traceback (most recent call last):
  File "<string>", line 1, in <module>
AttributeError: 'file' object has no attribute 'buffer'

should we do something like this:
```
if "buffer" in dir(sys.stdin):
  return sys.stdin.buffer # python3
else:
  return sys.stdin # python2
```

Reply:
@adrianreber are you going to fix this?

Reply:
I think it is. Although I also have trouble reading this here in GitHub, but line 14 contains: `if (sys.version_info < (3, 0)):`